_data/languages.yml

```diff
@@ -241,7 +241,7 @@
   territories: [ ro, ru, ua ]
 
 -
-  codes: [ cnr ]
+  codes: [ cnr, me ]
```

Comment:
I'd remove this, it's a Niutrans mistake afaict, our infra will handle it anyway.

Reply:
done